src/adapters/http/codec.ts

```diff
@@ -0,0 +1,20 @@
+import * as ts from 'io-ts';
+import * as PR from 'io-ts/PathReporter';
+import { Problem } from '../../generated/definitions/Problem';
+
+export const makeProblemFromErrors = (errors: ts.Errors): Problem => ({
```

Comment:
What do you think to rename this method?
```suggestion
export const makeBadInputProblem = (errors: ts.Errors): Problem => ({
```

Reply:
since _to_ and _from_ parts are self evident looking at the method typings, I suggest to cut the name ie.

```suggestion
export const makeProblemFromErrors = (errors: ts.Errors): Problem => ({
```
```suggestion
export const toProblem = (errors: ts.Errors): Problem => ({
```

or alternatively 
```suggestion
export const fromErrors = (errors: ts.Errors): Problem => ({
```

Reply:
note that it's the same naming as fp-ts converters (`fromEither`, ...). that's because the `to` part is implied by the module (and the typings)

Reply:
This is true. If we do that, should we also change the name of the `makeProblem` function, contained in the same file? Renaming it to `make`? At this point, we may also rename the file from `codec` to `Problem`

Reply:
`fromErrors`?

Reply:
The file [codec.ts](https://github.com/pagopa/pn-local-emulator-poc/pull/14/files#diff-276ddc9a9babf7b03bf624bbd8d20c0c1587067738bd9e1210514ea9a71f0d8a) contains two functions, one is `makeProblemFromErrors` and the other one is `makeProblem`. I like the idea of cutting the name keeping only the *from* part, but in my opinion, if we do that, we should also change the name of `makeProblem` function to `fromNumber`, and rename the file from `codec` to `Problem`.

Reply:
I agree!